### ISSUE TITLE
Installation Failed - both for Rolling and LTS 

### ISSUE THREAD

**Issue by x9xhack** [hosseinseilani/predator-os#9]
When I try to install the distros both rolling and lts release , I get a grub install error at the end . I am attaching screenshot

![image](https://github.com/hosseinseilani/predator-os/assets/10289716/944bd86e-3597-4586-9311-e7c6d50994cb)

**Comment by hosseinseilani** [comment]
hi dear .
this problem is fixed. and the new rebuild iso is on the site.
but you can fix it manually,
in live mode, before starting  calamares, just go to the path:
and remove  " /etc/grub.d/01_tuned "
and try again to install it.

**Comment by ErhanKarabulut** [comment]
Hi hossein ,

I removed " /etc/grub.d/01_tuned" and executed calamares but same error was exist in iso I downloaded before.

I downloaded new iso from website but in that one I checked " /etc/grub.d/01_tuned" file and it it still there . Additionally calamares doesnt work. Checked install system shell script and got this error.

06:17:07 [1]: void bail(const QString&, const QString&)
    ERROR: FATAL in "/etc/calamares/branding/eggs/branding.desc" 
     bad conversion
